extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java

```diff
@@ -467,6 +477,37 @@ public void setPrincipalClaim(String principalClaim) {
         }
     }
 
+    @ConfigGroup
+    public static class Proxy {
+
+        /**
+         * The host (name or IP address) of the Proxy.<br/>
+         * Note: If OIDC adapter needs to use a Proxy to talk with OIDC server (Provider),
+         * then at least both "host" and "port" items must be configured to enable the usage of a Proxy.
```

Comment:
`port` is no longer required now as it assumes 80 as the default value

Reply:
You're right! Again! 😊  Sorry, forgot about it.
I was in a hurry to make sure this small scope is covered.